api/app/Repositories/Interfaces/IUserRepository.php

```diff
@@ -11,7 +12,29 @@ interface IUserRepository
      *
      * @param string $apiKey user api key
      * @return User|null user if found, otherwise null.
-     * @throws \Exception in case of DB connection failure.
+     * @throws Exception in case of DB connection failure.
      */
     public function getUserByApiKey(string $apiKey) : ?User;
+    
+    /**
+     * Get all users.
+     *
+     * @return User[]|null
+     * @throws Exception in case of DB connection failure.
+     */
+    public function getUsers() : ?array;
```

Comment:
You should not return null instead of the array if it does not mean something serious like "feature is not supported", because it complicates further checks

Reply:
ok